load-tests/zoom_rio_de_janerio_with_contributor_filter.js

```diff
@@ -24,9 +24,14 @@ har.log.entries.forEach((entry) => {
     const cacheKey = crypto
         .sha1(entry.request.url.match(pattern)[5], "hex")
         .slice(0, 8);
+    // Including the __VU in the cache key should generate a worse situation
+    // than typical. Each batch of requests is made by a separate VU and when
+    // VU_MULTIPLIER is larger than 1, we repeat the same batch ofrequests a
+    // number of times equal to VU_MULTIPLIER. Using __VU in the cache key means
+    // subsequent requests for the same batch will not hit the cache.
     const url = entry.request.url.replace(
         pattern,
-        `${cacheKey}${__ENV.CACHE_KEY_SUFFIX}/$2/$3/$4.pbf?$5`
+        `${cacheKey}-vu-${__VU}-${__ENV.CACHE_KEY_SUFFIX}/$2/$3/$4.pbf?$5`
```

Comment:
It took me a while to figure out where `__VU` was defined: https://k6.io/docs/using-k6/execution-context-variables/#_vu-and-_iter-discouraged. Seemingly discouraged by the docs, but works as expected here.

Reply:
I noticed the discouragement, too, but continued using __VU since it was already used successfully elsewhere in the script.